os/include/tinyara/reboot_reason.h

```diff
@@ -40,6 +40,10 @@ typedef enum {
 	REBOOT_SYSTEM_BINARY_UPDATE        = 7,	 /* Reboot for Binary Update */
 	REBOOT_SYSTEM_BINARY_RECOVERYFAIL  = 8,	 /* Binary Recovery Fail */
 
+	REBOOT_SYSTEM_DSLP_RESET           = 27, /* System wake up from deep sleep */
+	REBOOT_SYSTEM_SYS_RESET            = 28, /* System reset by calling sys_reset */
+	REBOOT_SYSTEM_BOD_RESET            = 29, /* Brownout reset */
```

Comment:
Is there any case for calling `sys_reset` from bootloader?
Within TizenRT side, we all add the reboot reason when calling `sys_reset`.